components/choices/index.js

```diff
@@ -105,7 +111,40 @@ const updateChoiceById = async (req, response) => {
     if (total.eq(0)) {
       throw new Error("No balance at proposal level");
     }
+    const isVoted = await db_connect
```

Comment:
Are these changes relevant to this PR ?

Reply:
@Man-Jain I copied over some of the changes from the [original PR c](https://github.com/dOrgTech/homebase-lite-backend/pull/17/files#diff-51955b00ac2104e971f4966afbcbf92b6dab10645c827d0ceed8838410d90649)reated for this. Please do let me know what should I remove from here?